src/main/java/bio/terra/cli/command/cluster/Launch.java

```diff
@@ -0,0 +1,57 @@
+package bio.terra.cli.command.cluster;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Workspace;
+import bio.terra.cli.businessobject.resource.GcpDataprocCluster;
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.command.shared.options.DataprocClusterName;
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.command.shared.options.WorkspaceOverride;
+import bio.terra.cli.exception.UserActionableException;
+import bio.terra.cli.service.AxonServerService;
+import bio.terra.workspace.model.CloudPlatform;
+import java.util.UUID;
+import org.json.JSONObject;
+import picocli.CommandLine;
+
+/** This class corresponds to the third-level "terra cluster launch" command. */
+@CommandLine.Command(
```

Comment:
is dataproc launched?
if not, flag-guard these commands thanks.

i added a feature class [here](https://github.com/DataBiosphere/terra-cli/blob/d0473814b032b475f8f4cc0183f12da8b276c042/src/main/java/bio/terra/cli/service/FeatureService.java#L18).